service/models/person_model.rb

```diff
@@ -0,0 +1,8 @@
+class Person < Hoodoo::ActiveRecord::Base
+  # dating_enabled # Causes an issue running curl - "code": "platform.fault", "message": "PG::UndefinedTable: ERROR:  relation \"people_history_entries\" does not exist\nLINE 8:
```

Comment:
Thats because when you have `dating_enabled` it requires that you install a bunch of triggers etc in the database, which we haven't looked at for this challenge. Take a look at [automatic dating](http://loyaltynz.github.io/hoodoo/guides_0300_active_record.html#heading_number_39) for more details.